### PR TITLE
classcomp.test_submit_job_comp failed to find expected log message

### PR DESCRIPTION
**Pull request by dinesh1306**
#### Describe Bug or Feature
classcomp.test_submit_job_comp failed to find expected log message.
test expect following log message, but cannot get expected log message and test got failled.

#### Describe Your Change
respective logs are no more present in PBS. remove log match code from test.

#### Attach Test and Valgrind Logs/Output
before fix:
[res_before_fix.txt](https://github.com/openpbs/openpbs/files/4717282/res_before_fix.txt)
After fix:
[res_before_fix.txt](https://github.com/openpbs/openpbs/files/4717284/res_before_fix.txt)

### CODE REVIEW

File: test/tests/functional/pbs_compress.py [hunk 1]
```diff
@@ -60,8 +60,6 @@ def test_submit_job_comp(self):
         j.set_attributes({ATTR_v: "ABC=%s" % data})
         jid = self.server.submit(j)
         self.server.expect(JOB, {'job_state': 'R'}, id=jid)
-        self.server.log_match("tpp_deflate called inside tpp_send routine")
```
Comment (hirenvadalia) [inline test/tests/functional/pbs_compress.py:63]: @dinesh1306 Can you please change test suite name also? its not as per our PTL guidelines of test suite name. Thx!

**Comment by dinesh1306** [comment]
@hirenvadalia , I have incorporate review comment.
please take a look.
[res_TestCompression.txt](https://github.com/openpbs/openpbs/files/4721982/res_TestCompression.txt)

File: test/tests/functional/pbs_compress.py [hunk 2]
```diff
@@ -38,11 +38,11 @@
 # subject to Altair's trademark licensing policies.
 
 
+from tests.functional import *
 import os
-from ptl.utils.pbs_testsuite import *
 
 
-class classcomp(PBSTestSuite):
+class TestCompression(TestFunctional):
```
Comment (hirenvadalia) [inline test/tests/functional/pbs_compress.py:45]: Multiple thoughts on this test suite:
1. If we are not checking log to see whether it has compressed or not then why we need this test?
2. this is something tpp related test and we have one suite for it (IIRC, its TestTPP), can we move this test in that suite?

Comment (riyazhakki) [reply test/tests/functional/pbs_compress.py:45]: Yeah, I agree the purpose of the test was to match the log message use to appear after calling tpp_inflate and  tpp_deflate, now the log message is not present in the code so it is ok to remove this test.

Comment (dinesh1306) [reply test/tests/functional/pbs_compress.py:45]: thanks @riyazhakki. 
@bhroam & @hirenvadalia please let me know if you are fine with removing this test.

Comment (hirenvadalia) [reply test/tests/functional/pbs_compress.py:45]: I am ok with removal of this suite.

Comment (bhroam) [reply test/tests/functional/pbs_compress.py:45]: I'm fine with the removal of the test too.  It sounds like it is no longer needed.

**Comment by dinesh1306** [comment]
@hirenvadalia , @bhroam i have incorporate review comment. Please have a look.

**Pull request merged**